src/mo_interpreter/interpret.ml

```diff
@@ -620,10 +620,17 @@ and interpret_exp_mut env exp (k : V.value V.cont) =
     interpret_exp env exp1 (fun v1 ->
       interpret_cases env cases exp.at v1 k
       )
-  | TryE (exp1, cases) ->
-    let k' = fun v1 -> interpret_catches env cases exp.at v1 k in
-    let env' = { env with throws = Some k' } in
-    interpret_exp env' exp1 k
+  | TryE (exp1, cases, exp2_opt) ->
+    let k, env = match exp2_opt with
+      | None -> k, env
+      | Some exp2 ->
+        let pre k v = interpret_exp env exp2 (fun _ -> k v) in
```

Comment:
```suggestion
        let pre k v = interpret_exp env exp2 (fun v2 -> V.as_unit v2; k v) in
```
(Might be a good sanity check)

Reply:
I added a comment above to verify the evaluation result. But `V.as_unit` should suffice here.